recipes/cockroach/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+rocksdb:
+ - 6.2
+libprotobuf:
+ - 3.6.0
```

Comment:
I am pinning libprotobuf to 3.6.0 because our version of libprotobuf [does not include all the headers](https://github.com/conda-forge/libprotobuf-feedstock/issues/50) that cockroachdb expects in order to build. So I am using the headers of [libprotobuf 3.6.0] that is vendored in the source archive. 

I tried used Conda-forge's pinned version (3.11.0) with the headers from 3.6.0 and that just broke the build.